pkg/conf/conf.go

```diff
@@ -92,6 +92,24 @@ func LoadHarvestConfig(configPath string) error {
 	return nil
 }
 
+func SafeHarvestConfig(configPath string) error {
+	var (
+		contents []byte
+		err      error
+		fileName string
+	)
+	if !configRead {
+		panic("config not read")
+	}
+	if contents, err = yaml.Marshal(&Config); err != nil {
+		return err
+	}
+	if filepaths := strings.Split(configPath, "/"); len(filepaths) > 1 {
+		fileName = filepaths[1]
+	}
+	return ioutil.WriteFile("new_"+fileName, contents, 0644)
```

Comment:
What happens when multiple poller try to execute this same line? This code will have race condition.

Reply:
@Hardikl and I discussed this earlier and I don't think his code reflects what we discussed yet - the idea is each poller will create a payload file prefixed with the poller name, which is unique per harvest.yml

Reply:
@Hardikl  where are we using this file again? I see that poller will still read from default config which won't have uuid?

Reply:
this is removed as not needed.